pkg/systemd/services/services.scss

```diff
@@ -25,3 +25,17 @@
 .pf-c-data-list__item-action {
     min-width: 10rem;
 }
+
+// Make service IDs look like links
```

Comment:
If they should looks like links maybe they should be links?

Reply:
Yeah, perhaps they should be links. They'd still need a little extra boost to make them hover over the clickable area of each row, but it'd be an improvement.